client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -155,11 +155,11 @@ function CheckoutSummaryFeaturesList( props ) {
 
 	let refundText = translate( 'Money back guarantee' );
 	if ( hasDomainsInCart && ! hasPlanInCart ) {
-		refundText = translate( '4 day money back guarantee' );
+		refundText = translate( '4-day money back guarantee' );
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.